src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/TilbakekrevingClient.kt

```diff
@@ -0,0 +1,89 @@
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.web.client.RestOperations
+import org.springframework.web.util.UriComponentsBuilder
+import java.net.URI
+import java.util.UUID
+
+@Component
+class TilbakekrevingClient(@Qualifier("azure") restOperations: RestOperations,
+                           @Value("\${FAMILIE_TILBAKE_URL}") private val familieTilbakeUri: URI)
+    : AbstractRestClient(restOperations, "familie.tilbakekreving") {
+
+    private val hentForhåndsvisningVarselbrevUri: URI = UriComponentsBuilder.fromUri(familieTilbakeUri)
+            .pathSegment("api/dokument/forhandsvis-varselbrev")
+            .build()
+            .toUri()
+
+    private val opprettTilbakekrevingUri: URI =
+            UriComponentsBuilder.fromUri(familieTilbakeUri).pathSegment("api/behandling/v1").build().toUri()
+
+    private val opprettBehandlingManueltUri = UriComponentsBuilder.fromUri(familieTilbakeUri)
+            .pathSegment("api/behandling/manuelt/task/v1")
+            .build()
+            .toUri()
+
+    private fun finnesÅpenBehandlingUri(fagsakId: UUID) = UriComponentsBuilder.fromUri(familieTilbakeUri)
+            .pathSegment("api/fagsystem/${Fagsystem.EF}/fagsak/$fagsakId/finnesApenBehandling/v1")
+            .build()
+            .toUri()
+
+    private fun finnBehandlingerUri(fagsakId: UUID) = UriComponentsBuilder.fromUri(familieTilbakeUri)
+            .pathSegment("api/fagsystem/${Fagsystem.EF}/fagsak/$fagsakId/behandlinger/v1")
+            .build()
+            .toUri()
+
+    private fun kanBehandlingOpprettesManueltUri(fagsakId: UUID, ytelsestype: Ytelsestype) = UriComponentsBuilder.fromUri(
+            familieTilbakeUri)
+            .pathSegment("api/ytelsestype/$ytelsestype/fagsak/$fagsakId/kanBehandlingOpprettesManuelt/v1")
+            .build()
+            .toUri()
+
+    fun hentForhåndsvisningVarselbrev(forhåndsvisVarselbrevRequest: ForhåndsvisVarselbrevRequest): ByteArray {
+        return postForEntity(hentForhåndsvisningVarselbrevUri,
+                             forhåndsvisVarselbrevRequest,
+                             HttpHeaders().apply { accept = listOf(MediaType.APPLICATION_PDF) })
+    }
+
+    fun opprettBehandling(opprettTilbakekrevingRequest: OpprettTilbakekrevingRequest): String {
+        val response: Ressurs<String> = postForEntity(opprettTilbakekrevingUri, opprettTilbakekrevingRequest)
+        return response.getDataOrThrow()
+    }
+
+    fun opprettBehandlingManuelt(request: OpprettManueltTilbakekrevingRequest) {
+        postForEntity<Ressurs<String>>(opprettBehandlingManueltUri, request)
+    }
+
+    fun finnesÅpenBehandling(fagsakId: UUID): Boolean {
+        val response: Ressurs<FinnesBehandlingResponse> = getForEntity(finnesÅpenBehandlingUri(fagsakId))
+        return response.getDataOrThrow().finnesÅpenBehandling
+    }
+
+    fun finnBehandlinger(fagsakId: UUID): List<Behandling> {
```

Comment:
Trenger vi `finnBehandlinger` i iverksett, eller er det egentlige noe som ef-sak kun trenger? 
Eller kommer iverksett være en proxy for alle kall mot tilbake?